ph5/utilities/ph5validate.py

```diff
@@ -846,15 +829,25 @@ def get_args():
         help="Verbose logging.")
 
     args = parser.parse_args()
-
-    # Add formatter
-    args.level = str(args.level).upper()
-    if args.level != "ERROR" and args.level != "WARNING" and \
-            args.level != "INFO":
-        raise ValueError("Invalid logging level.")
-
+    outfile = args.outfile
+    PH5 = args.nickname
+    if not os.path.exists(PH5) and not os.path.exists(PH5 + '.ph5'):
+        LOGGER.error("{0} not found.".format(PH5))
+        sys.exit()
     if args.verbose is True:
         LOGGER.parent.handlers[0].setLevel(logging.DEBUG)
+    # Set up logging
+    # Write log to file
+    ch = logging.FileHandler(outfile)
+    level = args.level
+    if level == "ERROR":
+        ch.setLevel(logging.ERROR)
+    elif level == "WARNING":
+        ch.setLevel(logging.WARNING)
+    elif level == "INFO":
+        ch.setLevel(logging.INFO)
+    LOGGER.addHandler(ch)
+
```

Comment:
I'm about to remove adding file handler to logger from line 839 - 849 here because the content of outfile is created by write_to_log(), not by logger. Is this ok for me to do that?

Reply:
I think you should keep that section of code because that is where you create the handler and formatter for your log file and where you set the formatter and level of logging details.

Reply:
I will keep this code.